src/types/api.ts

```diff
@@ -486,6 +486,13 @@ export type SearchResponse = {
 export type AnnotationModeration = {
   moderation_status: ModerationStatus;
 
+  /**
+   * The annotation's current moderation status, used to know if the annotation
+   * has been moderated since it was first loaded, which would cause a conflict
```

Comment:
```suggestion
   * has been moderated since it was loaded, which would cause a conflict
```

The client always sends the latest moderation status it has, not always the "first" time the annotation it was loaded in a session.